cylc/flow/scheduler.py

```diff
@@ -1025,7 +1031,10 @@ def command_reload_workflow(self):
         pri_dao = self.workflow_db_mgr.get_pri_dao()
         pri_dao.select_workflow_params(self._load_workflow_params)
 
-        self.load_flow_file(is_reload=True)
+        try:
+            self.load_flow_file(is_reload=True)
+        except (ParsecError, CylcConfigError) as exc:
+            raise CommandFailedError(f"{type(exc).__name__}: {exc}")
```

Comment:
Why the re-raise here?

Reply:
That wouldn't work for `CylcConfigError`s

Reply:
Do we have a problem if `CylcConfigError`s are raised during reload?

Reply:
It would produce traceback when we surely only want a one-liner?

Reply:
Would have thought so.

Reply:
Resolved - https://github.com/cylc/cylc-flow/pull/4720#issuecomment-1084698176